src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -141,39 +168,51 @@ private protected override void SaveModel(ModelSaveContext ctx)
 
             base.SaveColumns(ctx);
             ctx.SaveStringOrNull(ImageFolder);
+            ctx.Writer.Write(UseImageType);
         }
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "IMGLOADR",
                 //verWrittenCur: 0x00010001, // Initial
-                verWrittenCur: 0x00010002, // Swith from OpenCV to Bitmap
-                verReadableCur: 0x00010002,
-                verWeCanReadBack: 0x00010002,
+                verWrittenCur: 0x00010003, // Added support for output type as byte array
+                verReadableCur: 0x00010003,
+                verWeCanReadBack: 0x00010003,
```

Comment:
>0x00010003 [](start = 34, length = 10)

This should still be 0x00010002, since you are taking care of backwards compatibility in line 138.